### DISCUSSION TITLE
Amazon fire  stick 2023

### DISCUSSION THREAD

**Discussion by flipper2222222** [GerryDazoo/SlingerPlayer#25] [Q&A]
hi all, 

might be too soon to ask but will the Slinger Player install and run new versions of fire stick just released ?

thanks

**Comment by GerryDazoo** [comment]
I would be surprised if it didn't. Do you know what version and API level of Android it's using?

**Comment by flipper2222222** [comment]
Fire OS 8 (Android 11)

the specs I could find.

https://developer.amazon.com/docs/fire-tv/device-specifications-fire-tv-streaming-media-player.html

**Comment by Humbug787** [comment]
Hi 
I have picked up the new Firestick and unfortunately Slinger Player runs badly. It seems to stutter and has major audio sync issues.  I have no idea of the Android API just wanted to let people know my experience.

Thanks

**Reply by GerryDazoo** [reply]
Have you tried some other players to see how they perform?

**Reply by Humbug787** [reply]
Yes, VLC works with a big change to the audio sync and MPlayer works well.  I love Slinger Player on my old Firestick though because the remote response time is so quick in comparison

**Reply by Humbug787** [reply]
Is there a possibility of a fix Gerry?

**Reply by GerryDazoo** [reply]
I have no way of testing any changes, even if I could figure out what might be going wrong. But what you're describing sounds like a networking problem. The remote works better with Slinger Player because it is configured for minimal buffering unlike VLC which buffers up multiple seconds of data. So if you have issues getting the bits to the player it will behave as you have described. 
I wonder if flipper2222222 bought one and got it going.

**Reply by flipper2222222** [reply]
hi everyone, I am going to buy one of the new Firestick Models on the Black Friday sales later this month, I will report back my findings. 
Humbug787, which model did you buy? I am planning on buying the 4K version. cheers

**Reply by Humbug787** [reply]
I went for the 4K Max for the 16gb of memory so I don’t have delete stuff to add stuff. A very good performer all round so far

**Reply by flipper2222222** [reply]
yes that model is available for sale here down under too, so will buy that one also and test, will update after then.

**Reply by rossfletcher** [reply]
Hi. can anyone confirm which Firestick Slingerplayer definitely works OK on please? I don't mind whether I but the 4K or the previous HD model - just want to get one or the other before the Black Friday deal disappears. many thanks.

**Comment by flipper2222222** [comment]
> Hi I have picked up the new Firestick and unfortunately Slinger Player runs badly. It seems to stutter and has major audio sync issues. I have no idea of the Android API just wanted to let people know my experience.
> 
> Thanks

@Humbug787 
did you get the stutter and audio sync sorted on your Firestick yet? thanks

**Reply by Humbug787** [reply]
Hi

Not really sorted no.  It stutters when you first start to play but that settles down, the audio is always slightly out of sync but maybe not as bad as I had first found. I still think that the ‘Gen 1’ Firestick was much better with Slinger Player but that would personally not stop me buying the Gen 2 as the rest of its performance is very good.

I hope that helps as I know you are just about to press the buy button on the Prime sales..

**Reply by flipper2222222** [reply]
thanks for the update, they just dropped 50% in price a few days ago so yes I may as well give it a go and report back cheers

**Reply by GerryDazoo** [reply]
You can adjust the audio/video sync in the Firestick settings.

**Reply by rossfletcher** [reply]
Thanks for those updates everyone. I just want a Firestick which will run the Slingerplayer apk successfully. I have Chromecast with Google TV at home. What I want is a streaming device which will run Slingerplayer when I am on holiday. It does not have to be 4K and it does not need to be capable of anything else. I have tried a 2016 model which I borrowed and it just displays a large Slingerplayer logo then closes down. GerryDazoo, could I ask which exact model of Firestick you wrote the Slingerplayer Firestick apk for and successfully tested it with please? Many thanks,

**Reply by GerryDazoo** [reply]
I've got 4 sticks, a plain ol' Firestick, two Firestick 4Ks and a 4K Max.

**Reply by rossfletcher** [reply]
That's great thanks.

**Comment by flipper2222222** [comment]
I just bought a Fire TV Stick 4K max and set it up. got slinger up and working as expected. the audio and video are in sync. I haven't tested it will away from home, just using it on my local network. 

hope this helps. 

cheers

**Reply by Humbug787** [reply]
That’s great news Flipper. I must have something wrong in my set up then. I will experiment with Gerry’s suggestion for the in built A/V Sync (Bouncy Ball Thingy) and see if I can get it sorted.

**Reply by rossfletcher** [reply]
Thanks flipper2222222. That is very helpful.

**Comment by flipper2222222** [comment]
I am wondering what sort of result I will get when viewing remotely from OS. has anyone used the firestick \slinger player  intercountry or intercontinental ?

**Reply by Humbug787** [reply]
As a regular traveller that is pretty much all I use mine for. I have 500mbps upload at home so that should never be a factor there. It has been very good with my Gen 1 Firestick, and I am hopeful that with a bit of tweaking I can get the Gen 2 working well too, still depends on the quality of the local hotel WiFi but overall it’s a winner.

**Reply by GerryDazoo** [reply]
I stream from Nova Scotia to southern Pennsylvania during the late summer. Works fine. Note, the automatic query for Slingboxes on your home network won't work so you'll have to type in the URLs manually.

**Reply by rossfletcher** [reply]
My upload speed is around 25 Mbps. I have used Slingerplayer from Android phone and Windows 10 laptop in other parts of the UK and from Majorca to my home in the UK over hotel WiFi and 4G phone connection. Streaming in HD has been faultless, even with NordVPN connected. Ordered my Fire Stick 4K Max now and if that works OK I'm made up. I really appreciate Gerry's efforts in making all this available. I was gutted when Slingbox announced the withdrawal of support, but now it all works better than ever.

**Reply by rossfletcher** [reply]
Fire Stick 4K Max worked straightaway on my home LAN and also with the Slingbox connected to my home LAN and the Fire Stick connected to my phone's hotspot with the phone connected via its data connection. Only issue I have is that the video is about five seconds behind the sound. Gerry suggested that this could be adjusted in the Fire Stick settings so I will try that tomorrow and report back.

**Reply by GerryDazoo** [reply]
5 seconds sounds like alot to me. Hmmm..

**Reply by GerryDazoo** [reply]
A quite power cycle of your Slingbox might help.

**Reply by rossfletcher** [reply]
Sorry for delay in replying. Was waiting to visit my son to test over the internet as phone's hotspot isn't great where I live. I power cycled the Slingbox as Gerry suggested and now everything with Slingplayer running on the Fire Stick 4K Max over the internet is fine, save a very small out of sync interval between video and sound. I can experiment with Fire Stick's AV sync then set it back to the default for other apps. I will also compare the Fire Stick Slingerplayer over the net to Slingerplayer from my Windows 10 laptop and see whether it is completely in sync on Windows over the same connection. Pretty close to perfect though. Thanks again Gerry.

**Comment by flipper2222222** [comment]
500! I wish, but doesn't the slingbox upload have a limit which is much less than 500mb?  shouldn't 25 meg upload be more than sufficient for slingbox\slinger operations?

and I have found local overseas network congestion or poor routing conditions between countries can severely affect the quality of the stream.

to try and get some more consistency and a more direct connection ,,,,I have bought a VPN router for my home and am going to try a direct vpn connection to my router and use my local lan address to the slinger, not sure if will be a panacea to the slinger \slingbox latency issues, but I thought worth a try. I will also use the vpn to give me a home country IP address to get around other geo blocking issues.

**Comment by flipper2222222** [comment]
Great news!

**Comment by flipper2222222** [comment]
just s small update. Viewing from Thailand to Australia, initial testing mostly ok, had a episode of stuttering which caused the delay to widen then a lock up of the app. I am going to setup a vpn from the viewing location to slingbox location and see iff this helps.

**Reply by GerryDazoo** [reply]
Flipper, checkout the latest binary. I've added Play/Pause button support. Pausing for a second or two will build up a jitter buffer which should behave better with a link with high latencies. I have no way of testing this myself so please let us know if it improves your experience.
Cavet: The buffering is happening in memory so don't pause for a long time or "bad things" might happen.
I don't think a VPN will help. The encrypted VPN bits are still using the same global Internet pipes.

**Reply by flipper2222222** [reply]
thanks for the update Gerry that is a nice little addition, I will check it out, is there a new version number with this release?

My thoughts behind the VPN is that it make more of a direct connection in between the 2 points and make my gateway in Australia and somehow overcome packet loss from traversing fewer routers, or is that incorrect?

**Reply by flipper2222222** [reply]
I just installed the Update SlingerPlayer.apk f385c25   5 days ago
but I only see the play full screen and the A/R button,
am I missing something?
I uninstalled the previous version first
thanks

**Reply by flipper2222222** [reply]
further to my comment on 28/12/23
I often found that the video would free but the audio continued. to fix this I would pause the video  and force stop  the task in settings, app.
